Bottom: 7f5e34480dc7381beb32fff528d5cecdf2f13fc6
Top:    3fd510e9ab6e34c2c953080f7c58f84af3b7436f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-04 15:57:59 +0000

Adjust circular approximation


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..2d65302 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
